Reply/app/src/main/res/values/themes.xml

```diff
@@ -19,8 +19,11 @@
 
     <style name="Theme.Reply" parent="Theme.Material3.DayNight.NoActionBar">
 
+	      <!--Transparent Navigation Bar and StatusBar-->
+	      <item name="android:statusBarColor">@android:color/transparent</item>
```

Comment:
Fix indentation.

Also, let's put these android specific attrs after the color slot assignments - after `scrimBackground` where they where.